x-pack/plugins/apm/public/components/app/settings/agent_explorer/index.tsx

```diff
@@ -201,6 +201,7 @@ export function AgentExplorer() {
           isLoading={isLoading}
           items={agents.data?.items ?? []}
           noItemsMessage={noItemsMessage}
+          latestVersionsFailed={!!agents.data?.latestVersionError}
```

Comment:
What about passing the error down so you can show the status code to the user?

```suggestion
          latestVersionError={agents.data?.latestVersionError}
```

Reply:
Okay, that's fine.

Reply:
Okay, that's fine.